### DISCUSSION TITLE
Option to disable Close on zip extractor

### DISCUSSION THREAD

**Discussion by ConnectAutomatisering** [ravibpatel/AutoUpdater.NET#704] [Ideas]
Hi,

Sometimes somebody closes the ZipExtractor while its unzipping. Thats not very handy of course because then the program wont start. Would it be possible to disable this?
![image](https://github.com/user-attachments/assets/f3cc94b0-3c20-44dc-8580-b857b21e98fc)

**Comment by ravibpatel** [comment]
I am disabling the close button in next version.
